pal/screen-adapter/web/screen-adapter.ts

```diff
@@ -474,8 +481,13 @@ class ScreenAdapter extends EventTarget {
                 }
                 notifyOrientationChange(tmpOrientation);
             };
-            mediaQueryPortrait.addEventListener('change', orientationChangeCallback);
-            mediaQueryLandscape.addEventListener('change', orientationChangeCallback);
+            if (mediaQueryPortrait.addEventListener) {
+                mediaQueryPortrait.addEventListener('change', orientationChangeCallback);
+                mediaQueryLandscape.addEventListener('change', orientationChangeCallback);
```

Comment:
Should also check it too?

Reply:
mediaQueryLandscape and mediaQueryPortrait are similar objects, so checking one of them is sufficien